c2rust-ast-exporter/Cargo.toml

```diff
@@ -21,10 +21,10 @@ serde_cbor = "0.11"
 
 [build-dependencies]
 bindgen = { version = "0.60", features = ["logging"] }
-clang-sys = "1.3"
+clang-sys = "1.4"
 # Pinning until https://github.com/rust-lang/cmake-rs/issues/131 is resolved
 # Fixed by https://github.com/rust-lang/cmake-rs/pull/146 on 5/12/2022; waiting for next release.
-cmake = "=0.1.45"
+cmake = "=0.1.48"
```

Comment:
The comment and pinning are there for a reason.  We can't upgrade until the next `cmake` version comes out, as there hasn't been a release since https://github.com/rust-lang/cmake-rs/pull/146 merged, fixing https://github.com/rust-lang/cmake-rs/issues/131 in `master`.

The `cmake` on Ubuntu 18 is an older version that doesn't support `--parallel`, so it fails there.  See the [CI failure](https://dev.azure.com/immunant/c2rust/_build/results?buildId=2693&view=logs&j=ae6377c6-aed8-50bb-2c16-24013b3441a6&t=19222e3a-3fea-5cc3-b08a-6873fb35b846&l=338).